source-code/core/src/lint/rules/additionalKey.ts

```diff
@@ -0,0 +1,24 @@
+import type { Context } from '../context.js'
+import { createRule } from '../rule.js'
+
+export const additionalKeyRule = createRule('inlang.additionalKey', () => {
+	let context: Context
+	let referenceLanguage: string
+
+	return {
+		initialize: (config) => {
```

Comment:
I am confused what `config` refers to.

I thought the inlang config is referred to but the config object seems to be a custom lint "config" object. Calling this `args` but be better whereas args is `{ env, context }`

Reply:
Forgotto rename this. Thanks!